athena/modules/text/module_text_llm/module_text_llm/approach_config.py

```
@@ -1,23 +1,19 @@
-from pathlib import Path
-from llm_core.loaders.llm_config_loader import get_llm_config
 from pydantic import BaseModel, Field
+from typing import Optional
+from pydantic import ConfigDict, BaseModel, Field
```

Ruff (0.13.3)

Redefinition of unused `BaseModel` from line 1

Remove definition: `BaseModel`

(F811)